tests/async/test_async_related_managers.py

```diff
@@ -0,0 +1,27 @@
+from django.test import TestCase
+
+from .models import ManyToManyModel
+
+
+class AsyncRelatedManagersOperationTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.mtm1 = ManyToManyModel.objects.create()
+
+    async def test_acreate(self):
+        await self.mtm1.simples.acreate(field=2)
+        refetched = await ManyToManyModel.objects.aget(pk=self.mtm1.pk)
```

Comment:
Do we need to refetch? :thinking:

Reply:
Tests pass without it, I'll remove to simplify